include/aie/AIEPasses.td

```diff
@@ -175,6 +175,19 @@ def AIERoutePathfinderFlows : Pass<"aie-create-pathfinder-flows", "ModuleOp"> {
   ];
 }
 
+def AIEMulticast : Pass<"aie-create-multicast", "ModuleOp"> {
+  let summary = "Replace multicast flow by flow";
+  let description = [{
+    An abstraction of multicast flow  
```

Comment:
This could be a bit more descriptive, maybe something like "This pass replaces AIE.multicast with the equivalent number of AIE.flow operations."